Makefile

```
@@ -219,6 +219,12 @@ import_prod_data:
 	@echo "🥫 Removing old archive in case you have one"
 	( rm -f ./html/data/openfoodfacts-mongodbdump.gz || true ) && ( rm -f ./html/data/gz-sha256sum || true )
 	@echo "🥫 Downloading full MongoDB dump from production …"
+# verify we got sufficient space, NEEDED is in octet, LEFT in ko, we normalize to MB and NEEDED is multiplied by two (because it also will be imported)
+	NEEDED=$$(curl -s --head https://static.openfoodfacts.org/data/openfoodfacts-mongodbdump.gz|grep -i content-length |cut -d ":" -f 2|tr -d " \r\n\t"); \
```

```suggestion
	NEEDED=$$(curl -s --head https://static.openfoodfacts.org/data/openfoodfacts-mongodbdump.gz|grep -i content-length: |cut -d ":" -f 2|tr -d " \r\n\t"); \
```

otherwise the grep also grabs another line

content-length: 8862170360
access-control-expose-headers: Content-Length,Content-Range

---

Thanks !